app/models/pawn.rb

```diff
@@ -21,17 +21,17 @@ def valid_move?(new_row, new_column)
       #check to see if there is a piece on the spot one ahead, regardless of what move (including move = 0)
       @one_ahead = Game.find(game_id).pieces.where(:position_row => position_row - 1, :position_column => position_column)
       #Only return true if we don't have a piece on the spot straight ahead
-      return true if new_row == position_row - 1 && new_column = position_column && @one_ahead.size == 0 #Patrick added secound new_column == position_column logic via Sam 
+      return true if new_row == position_row - 1 && new_column == position_column && @one_ahead.size == 0 #Patrick added secound new_column == position_column logic via Sam 
       
       #we capture things diagonally, first figure out if a piece of the opposing color is diagonal
       #If the piece is the same color, it won't be a valid move
```

Comment:
Does 'White Piece Logic' refer to the logic for capturing the white pieces? If not, then it comes off as confusing since lines 12-36 refer to black pieces.